polaris-react/src/utilities/frame/types.ts

```diff
@@ -40,7 +40,7 @@ export interface ContextualSaveBarProps {
   /** Accepts a string of content that will be rendered to the left of the actions */
   message?: string;
   /** Save or commit contextual save bar action with text defaulting to 'Save' */
-  saveAction?: ContextualSaveBarAction;
+  saveAction?: ContextualSaveBarAction | React.JSX.Element;
```

Comment:
(ReactNode is too wide of a net so this is more specific, as we only want to accept JSX elements not strings etc)